pkgs/development/haskell-modules/patches/ats-format.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/Setup.hs b/Setup.hs
```

Comment:
In general, we try to not carry around patches in nixpkgs unless absolutely necessary.

Would it be possible to instead get this fixed upstream?  (Possibly by adding a flag to disable manpage creation?)

Or, maybe you could just delete the `Setup.hs` file at the apropriate location, so the default one is used.

Also, why error are you getting when `Setup configure` is running?  Maybe you just need to create a `man` directory for it to use?

Reply:
It tries to write manpages to `$HOME/.local/share/man`, which doesn't exist. I could set `$HOME` to `/tmp` but I think it's too hacky.
I think I'll add a cabal flag and submit a PR to the upstream then.

Reply:
@poscat0x04 Does it make sense to just set `$HOME` to `$out`?  I could imagine that some users would actually want the man pages.

Reply:
Hmm, it seems that the cabal file is read after Setup.hs is being compiled so the CPP options are being ignored. Is it possible to let nix pass additional arguments (-optP) to `runghc`?